### ISSUE TITLE
Upload your add-on package: manifest.json was not found

### ISSUE THREAD

**Issue by borbiuk** [mozilla/addons-linter#5142]
### Describe the problem and steps to reproduce it:

I have already uploaded my extension to the Google Chrome Web Store. I use a `.sh` script in my extension repo to create a `.zip` file - [see](https://github.com/borbiuk/band-play/blob/master/publish.sh).

When I try to upload my add-on package I face an Error:
![image](https://github.com/mozilla/addons-linter/assets/41112208/19827a45-ff79-426f-bbae-00146d330218)

### What happened?

The `manifest.json` file was not found during the upload of the add-on package.

### What did you expect to happen?

I expected the package to be successfully uploaded.

### Anything else we should know?

.

```[tasklist]
### Tasks
```

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/ADDLINT-409)

**Comment by Rob--W** [comment]
You should zip the contents of the zip file, not the directory itself.

To fix it, change your build script from:

```sh
# Create a .zip file
zip -r "$zip_filename" "$temp_dir"
```

to

```
# Enter directory containing the built extension source.
cd "$temp_dir"

# Create a .zip file
zip -r "../$zip_filename" .

# Return to where we came from.
cd ..
```

NOTE: this assumes that `$temp_dir` is one directory level deep, which is the case in your build script. The generic solution (when `temp_dir` and/or the zip location are not one directory apart) is to use: `pushd "$temp_dir"` ; `zip -r "<absolute path to zip file>" *` ; `popd`
